modules/configuring-openshift-ingress-shards.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: PROCEDURE
+[id="configuring-openshift-ingress-shards_{context}"]
+= Configuring OpenShift ingress shards
+
+Before configuring Serving, you need to configure OpenShift ingress shards.
+
+.Procedure
+
+* In the `IngressController` CR, configure {ServerlessProductName} to match specific ingress shards with different domains with a label selector:
```

Comment:
is label selector being used to configure the {ServerlessProductName}? If so, I'd shuffle the sentence to put the two items closer toghether:
"Use a label selector in the `IngressController` CR to configure {ServerlessProductName} to match specific ingress shards with different domains"